app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/step/survey/TerminationSurveyDestination.kt

```diff
@@ -102,174 +133,332 @@ private fun TerminationSurveyScreen(
+        onValueChange = {
+          if (it.text.length <= 140) {
+            textValue = it
+          } else {
+            textValue = TextFieldValue(it.text.substring(0, 140))
+          }
+        },
+        cursorBrush = SolidColor(MaterialTheme.colorScheme.onSurface),
+        modifier = Modifier
+          .weight(1f)
+          .focusRequester(focusRequester)
+          .background(
+            MaterialTheme.colorScheme.surface,
+            shape = HedvigTextFieldDefaults.shape,
+          ),
+        textStyle = MaterialTheme.typography.bodyLarge.copy(color = MaterialTheme.colorScheme.onSurface),
+        decorationBox = @Composable { innerTextField ->
+          Column {
+            Row(
+              verticalAlignment = Alignment.Top,
+              horizontalArrangement = Arrangement.Start,
+              modifier = Modifier.weight(1f),
+            ) {
+              HedvigTextFieldDefaults.DecorationBox(
+                value = textValue.text,
+                colors = HedvigTextFieldDefaults.colors(
+                  typingHighlightColor = Color.Transparent,
+                  focusedContainerColor = Color.Transparent,
+                  unfocusedContainerColor = Color.Transparent,
+                  focusedIndicatorColor = MaterialTheme.colorScheme.onSurface,
+                  unfocusedIndicatorColor = MaterialTheme.colorScheme.onSurface,
+                ),
+                placeholder = {
+                  Text(
+                    text = hint,
+                    Modifier.fillMaxSize(),
+                    fontSize = MaterialTheme.typography.bodyLarge.fontSize,
+                  )
+                },
+                innerTextField = innerTextField,
+                enabled = true,
+                singleLine = false,
+                interactionSource = remember { MutableInteractionSource() },
+                visualTransformation = VisualTransformation.None,
+              )
+            }
+            Row(
+              horizontalArrangement = Arrangement.End,
+              modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 16.dp, end = 16.dp, bottom = 12.dp),
+            ) {
+              Text(
+                text = "${textValue.text.length}/140",
+                style = MaterialTheme.typography.titleSmall,
+                color = MaterialTheme.colorScheme.onSurfaceVariant,
+              )
+            }
+          }
+        },
+      )
+      Spacer(modifier = Modifier.height(8.dp))
+      Row(
+        modifier = Modifier
+          .fillMaxWidth(),
+        horizontalArrangement = Arrangement.Center,
+      ) {
+        HedvigContainedSmallButton(
+          text = stringResource(id = R.string.general_cancel_button),
+          onClick = {
+            focusRequester.freeFocus()
+            onCancelClick()
+          },
+          modifier = Modifier.weight(1f),
+          colors = ButtonDefaults.buttonColors(
+            containerColor = Color.Transparent,
```

Comment:
I think there was some color here in Figma. It is using the new design though, so if it's just too complex it's fine this way too for now.